### PR TITLE
Merge in V12 hotfix + cleanup

### PR DESCRIPTION
**Pull request by agritheory**

### CODE REVIEW

File: erpnext/accounts/party.py [hunk 1]
```diff
@@ -48,12 +50,13 @@ def _get_party_details(party=None, account=None, party_type="Customer", company=
...
 	ship_to = frappe.get_doc(party_type, ship_to) if ship_to else frappe._dict()
 	if ship_to:
-		out['ship_to_name'] = ship_to.get(scrub(party_type) + "_name")
+		party_details['ship_to_name'] = ship_to.get(scrub(party_type) + "_name")
 
-	party_address, shipping_address = set_address_details(out, party, party_type, doctype, company, party_address, shipping_address, ship_to)
-	set_contact_details(out, party, party_type)
-	set_other_values(out, party, party_type)
-	set_price_list(out, party, party_type, price_list, pos_profile)
+	print(party_details)
```
Comment (SaiFi0102) [inline erpnext/accounts/party.py]: Remove print

File: erpnext/accounts/party.py [hunk 2]
```diff
@@ -74,15 +77,15 @@ def _get_party_details(party=None, account=None, party_type="Customer", company=
 		party_details["sales_team"] = [{
 			"sales_person": d.sales_person,
 			"allocated_percentage": d.allocated_percentage or None
-		} for d in ship_to.get("sales_team") or party.get("sales_team")]
+		} for d in party.get("sales_team")]
```
Comment (SaiFi0102) [inline erpnext/accounts/party.py]: Why are you removing ship_to functionality?

Comment (agritheory) [reply erpnext/accounts/party.py]: Resolved with ed931e5

File: erpnext/accounts/party.py (hunk 1 above)
Comment (agritheory) [reply erpnext/accounts/party.py]: Resolved with ed931e5

**Pull request merged**